akka-docs/src/test/java/jdocs/stream/GraphStageDocTest.java

```diff
@@ -612,8 +614,8 @@ public void onPull() throws Exception {
           });
         }
       };
-
-      return new Tuple2(logic, promise.future());
+      
+      return new Pair<>(logic, FutureConverters.toJava(promise.future()));
```

Comment:
Not having the diamond in there makes it raw Tuple2 and just gives a warning but compiles?